server/api/controllers/memberships/create-membership.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  friendlyName: "Create memberships",
+
+  description: "Create a member record",
+
+  inputs: {
+    officialName: {
+      type: "string",
+    },
+    userId: {
+      type: "string",
+    },
+    classAttendance: {
+      type: "json",
+    },
+    recognitionDate: {
+      type: "string",
+    },
+    recommitmentDate: {
+      type: "string",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "Membership record created successfully",
+    },
+    invalid: {
+      responseType: "badRequest",
+      description: "Failed to create membership record",
+    },
+
+    duplicateRecords: {
```

Comment:
Is this being used?

Reply:
not being used. actually it only checks for one-on-one relation right now. so it wont have duplicates i think